### ISSUE TITLE
Don't encode query parameter examples

### ISSUE THREAD

**Issue by cmeeren** [Redocly/redoc#1138]
Consider the following query parameter definition:

```yaml
in: query
name: "test"
schema:
  type: string
  format: date-time
  example: "2019-07-03T15:03:00+02:00"
```

It's rendered like this:

![image](https://user-images.githubusercontent.com/7766733/70607104-22efb780-1bfe-11ea-9466-896aea3b0cf5.png)

I get that query parameter values must be suitably encoded when sending actual requests, but that should be handled by tooling. **The documentation should focus on readability**, and the example as rendered above is very hard to read and understand.

Therefore, I suggest *not* encoding query parameter examples. If the developer wishes to document encoded values, they are of course free to do that explicitly. Currently it's not possible to document unencoded values.

In other words, the example above should be rendered as `2019-07-03T15:03:00+02:00`, not `2019-07-03T15%3A03%3A00%2B02%3A00`.

Alternatively this behaviour could be configurable, but I'm happy if this is simply disabled and the examples rendered as-is.

**Comment by RomanHotsiy** [comment]
I agree. I will decode parameter examples in UI.
